src/test/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSFilterTest.java

```diff
@@ -29,9 +29,9 @@
 
 public class PSFilterTest extends CommandLineProgramTest {
 
-    private final String BWA_IMAGE_PATH = "src/test/resources/" + PSFilter.class.getPackage().getName().replace(".", "/") + "/hg19mini.fasta.bwa_image";
+    private final String BWA_IMAGE_PATH = "src/test/resources/" + PSFilter.class.getPackage().getName().replace(".", "/") + "/hg19mini.fasta.img";
 
-    final static List<List<String>> readFastq(final File fastqFile) {
```

Comment:
Could you use SVFastqUtils method to deal with Fastq formatted files? Otherwise is there something in htsjdk to handle Fastq?

Reply:
Now using htsjdk fastqreader